packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -45,7 +45,10 @@ import BottomSheetSubSheet from './sub-sheet';
 import NavigationHeader from './navigation-header';
 import { BottomSheetProvider } from './bottom-sheet-context';
 
-const DEFAULT_LAYOUT_ANIMATION = LayoutAnimation.Presets.easeInEaseOut;
+const DEFAULT_LAYOUT_ANIMATION_PROPERTIES = {
```

Comment:
See the comment in the description regarding this change. 
The issue only affects Android but code path to the default animation is only followed by Android. 
iOS is following the keyboard specific animation setup ( around line 123 in this diff ).

Regarding that, I did try to force Android into using the `keyboard` animation type. The RN docs do not say that that animation type is iOS only but I kept running into this issue https://github.com/facebook/react-native/issues/27839  ( I too confirmed that `UIManager.setLayoutAnimationEnabledExperimental(true)` is set up correctly.